.github/workflows/pull-request.yml

```diff
@@ -174,6 +174,8 @@ jobs:
             id: 13
           - name: "upload-download-actions"
             id: 14
+          - name: "delete-actions"
```

Comment:
can you please check this. there should be favorites-action e2e to be named.

Reply:
Also on line 106 we can remove -"favoriteActions".

Reply:
Ah yes, added delete too as Adam merged it yesterday